app/models/user.rb

```diff
@@ -16,6 +16,7 @@
 #  epr_uuid               :string(255)
 #
 class User < ApplicationRecord
+  strip_attributes only: %i[first_name last_name email epr_uuid]
```

Comment:
missing model specs https://github.com/rmm5t/strip_attributes#writing-tests the rest look good